pkg/hintrunner/zero/zerohint_blake_test.go

```diff
@@ -427,5 +428,98 @@ func TestZeroHintBlake(t *testing.T) {
+						feltUint64(3491828193), feltUint64(2085238082), feltUint64(1219908895), feltUint64(514171180),
+						feltUint64(4245497115), feltUint64(4193177630), feltUint64(1795745351), feltUint64(3144134277),
+						feltUint64(1013904242), feltUint64(2773480762), feltUint64(1359893119), feltUint64(2600822924),
+						feltUint64(528734635), feltUint64(1541459225), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(4294967295),
+						feltUint64(813310313), feltUint64(2491453561), feltUint64(3491828193), feltUint64(2085238082),
+						feltUint64(1219908895), feltUint64(514171180), feltUint64(4245497115), feltUint64(4193177630),
+						feltUint64(1795745351), feltUint64(3144134277), feltUint64(1013904242), feltUint64(2773480762),
+						feltUint64(1359893119), feltUint64(2600822924), feltUint64(528734635), feltUint64(1541459225),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(4294967295), feltUint64(813310313), feltUint64(2491453561),
+						feltUint64(3491828193), feltUint64(2085238082), feltUint64(1219908895), feltUint64(514171180),
+						feltUint64(4245497115), feltUint64(4193177630), feltUint64(1795745351), feltUint64(3144134277),
+						feltUint64(1013904242), feltUint64(2773480762), feltUint64(1359893119), feltUint64(2600822924),
+						feltUint64(528734635), feltUint64(1541459225), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(4294967295),
+						feltUint64(813310313), feltUint64(2491453561), feltUint64(3491828193), feltUint64(2085238082),
+						feltUint64(1219908895), feltUint64(514171180), feltUint64(4245497115), feltUint64(4193177630),
+					}),
+			},
+		},
+		"Blake2sFinalize check assert N_PACKED_INSTANCES": {
+			{
+				operanders: []*hintOperander{
+					{Name: "blake2s_ptr_end", Kind: apRelative, Value: addrWithSegment(1, 7)},
+					{Name: "N_PACKED_INSTANCES", Kind: fpRelative, Value: feltUint64(20)},
+					{Name: "INPUT_BLOCK_FELTS", Kind: apRelative, Value: feltUint64(16)},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newBlake2sFinalizeHint(ctx.operanders["blake2s_ptr_end"], ctx.operanders["N_PACKED_INSTANCES"], ctx.operanders["INPUT_BLOCK_FELTS"])
+				},
+				errCheck: errorTextContains(fmt.Sprintf("in range %s, got %s", "[0, 20)", "20")),
```

Comment:
Also in this errorTextContains I meant you can just go for:
```errCheck: errorTextContains(fmt.Sprintf("n_packed_instances should be in range [0, 20), got %d", 20))```
since the error message is hardcoded right?